products/health-care/digital-health-modernization/product/roadmap.md

```diff
@@ -12,9 +18,10 @@ Items in Now are not strongly order-dependent.
 - Tests hypothesis that modernization = action over understanding. 
 - Impacts public website and authenticated experience crews. 
 
-### Migrate MHV accounts into ID.ME or Login.gov accounts. 
+### Migrate MHV accounts into an appropriate single sign-on solution. 
```

Comment:
On MHV side:
- improve design and content around secure sign in AND/OR 
- remove option to log into MHV **without** secure sign-in (e.g. use only one button to sign in with MHV credentials)

On VA.gov side 
- improve design and content around secure sign-in